src/main/java/net/masterthought/cucumber/json/Element.java

```diff
@@ -126,6 +126,9 @@ private Status calculateStepsStatus() {
         StatusCounter statusCounter = new StatusCounter();
         for (Step step : steps) {
             Result result = step.getResult();
+            if (result == null) {
+                result = new Result();
+            }
```

Comment:
According to https://github.com/damianszczepanik/cucumber-reporting/commit/7d9b8c49e209d1a2d05937a5f4a4cd55cce63e0c#diff-1eeb2da981684473c263089457fbbca9R16 when this may happen?